doc/Quick-Start-Guide.rst

```diff
@@ -78,14 +86,37 @@ To build RPMs, run::
 
     make rpms
 
-if Makefile was generated already (in case you run ``scripts/m0 make`` from the previous steps), or run::
+if Makefile was generated already
+(in case you run ``scripts/m0 make`` from the previous steps), or run::
 
     ./autogen.sh && ./configure && make rpms
 
 to build from scratch.
 
 The generated RPMs will be placed at ``$HOME/rpmbuild/RPMS/$(arch)/`` directory.
 
+DEBs Packages Generation (Experimental)
+---------------------------------------
+
+To build .deb packages for Ubuntu or Debian,
+make sure the dependencies are installed 1st::
+
+    sudo apt install equivs # enable mk-build-deps
+    sudo mk-build-deps --install debian/control
+
+Then just run::
+
+    make deb
+
+if Makefile was generated already
+(in case you run ``scripts/m0 make`` from the previous steps), or run::
+
+    ./autogen.sh && ./configure --with-user-mode-only && make deb
```

Comment:
Can we simplify  it to following,

For building the motr repo,
/autogen.sh && ./configure --with-user-mode-only
./scripts/m0 make

And for building the motr packages,
For Redhat based OSes,
  use `make rpms`
and for Debian based OSes,
  use `make deb`

Reply:
Check now.